src/library/dao/cue.cpp

```diff
@@ -7,6 +7,8 @@
 #include "library/dao/cue.h"
 #include "util/assert.h"
 
+const QString defaultColor = "#FF0000";
```

Comment:
Should be enclosed in an anonymous namespace to restrict the visibility of this symbol. Otherwise it is visible globally and might cause conflicts during linking.